src/routes/(repl)/+layout.server.ts

```diff
@@ -34,7 +34,10 @@ export const load: LayoutServerLoad = async ({ params, locals, url, cookies }) =
 	let owner_id;
 	try {
 		const record = await get_repl_from_id(repl, locals.pocketbase);
-		files = record.files;
+		// i don't like this but the required parsing introduces a lot of types
+		// shit and i feel like it would be a pain to mantain...we have to
+		// leave a bit of typesafety here for a safe management
+		files = record.files as FileSystemTree;
```

Comment:
This ain't the end of the world.

If we start feeling we have to cheat TypeScript super often, we can think about a refactor, but I think this is more than fine for now.